### DISCUSSION TITLE
Lombok usage

### DISCUSSION THREAD

**Discussion by rogeriofrsouza** [nvim-java/nvim-java#94] [General]
Hello 👾

I just read this [previous discussion](https://github.com/orgs/nvim-java/discussions/82) about `Lombok` usage.

Unfortunately I'm facing the same issues even though I have all plugins updated. The `LSP` cannot recognize `Lombok` annotations and is issuing these errors:
![image](https://github.com/nvim-java/nvim-java/assets/77423511/dc49f011-d429-4f66-bed9-bcefc1b97d1f)

I'm using `Lazy.vim` here and followed the [installation steps](https://github.com/nvim-java/nvim-java/wiki/Lazyvim)
There is also Google Java Format and Checkstyle set on none-ls, but I don't know if these are related.

**Comment by s1n7ax** [comment]
You should have lombok library as a project dependency as well.

**Comment by rogeriofrsouza** [comment]
Seems like I forgot to execute `mvn dependency:resolve` 😅
It's working now, thank you very much!
